### ISSUE TITLE
Subdomain serving Bitcoin default mempool data unresponsive

### ISSUE THREAD

**Issue by adlai** [jhoenicke/mempool#76]
I'm posting this publicly for convenience, and so that folks can see that the problem has already been reported.

Going to the main site [e.g. [here](https://jochen-hoenicke.de/queue/#BTC,1w,fee)] and then clicking `BTC (default mempool)` causes the requests to go through a separate subdomain, and they currently fail. Ideally, such failure could simply make the button for the corresponding data type unclickable, although right now the quick fix is probably either deleting that one button, or cranking at whatever has caused the electrum subdomain to go quiet.

**Comment by jhoenicke** [comment]
I'm not sure what the problem was you were encountering.  For me the electrum subdomain is responsive and I tried both via ipv4 and ipv6.  Also since the statistics does not have any visible holes, I know it had been connected to the internet all the time.

I had a few problems that caused a complete system freeze three times between end of January to mid February, but they have been solved by a hardware replacement.

Maybe it was a temporary DNS problem?

There is a small probability that you still had the old mempool.js in cache, pointing you to core.jochen-hoenicke.de, which is down since September.  Or you are on some URL that I'm not aware of still existing.  However, the URL you included is working for me.  Currently supported URLs for the main page are:

- [mempool.jhoenicke.de](https://mempool.jhoenicke.de)  (preferred domain)
- jhoenicke.de/queue
- jochen-hoenicke.de/queue
- bitcoinqueue.com
